src/Symfony/Component/Serializer/Mapping/Loader/XmlFileLoader.php

```diff
@@ -70,6 +70,12 @@ public function loadClassMetadata(ClassMetadataInterface $classMetadata)
                 if (isset($attribute['serialized-name'])) {
                     $attributeMetadata->setSerializedName((string) $attribute['serialized-name']);
                 }
+
+                if (isset($attribute->{'embed-property'})) {
```

Comment:
Typo, here and below, you try to access array like object property.

Reply:
`$attribute` is a SimpleXMLElement, so it's seem the way for access to a node with an hyphen in it's key.

However, after verification, there is no need to check if it is set, like for Group :  https://github.com/symfony/symfony/blob/e30ad2181369b87ff61af8c377adbd65ac885b6e/src/Symfony/Component/Serializer/Mapping/Loader/XmlFileLoader.php#L62-L66